.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    parser: "@typescript-eslint/parser", // Specifies the ESLint parser
+    extends: [
+        "plugin:@typescript-eslint/recommended", // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+        "prettier/@typescript-eslint", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+        "plugin:prettier/recommended" // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    ],
+    parserOptions: {
+        ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
+        sourceType: "module" // Allows for the use of imports
+    },
+    rules: {
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
Yeah, I think this is necessary too, as not all libraries have types and so far there aren't any custom type definitions in the project.

Reply:
Agree. This could also be a `"warn"` so e.g. CI won't fail on it but the developer is notified.